File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/cpp/ADV-08_R.cpp
#include <cstdio>
#include <cstdint>
#include <iostream>
#include <string>

int main() {
    printf("ADV-START\n"); fflush(stdout);
    std::string l1, l2;
    std::getline(std::cin, l1);
    std::getline(std::cin, l2);
    std::int64_t a = std::stoll(l1);
    std::int64_t b = std::stoll(l2);
    std::int64_t q = a / b;
    printf("OBS=QUOT:%lld\n", (long long)q); fflush(stdout);
    printf("ADV-END\n"); fflush(stdout);
    return 0;
}

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/typescript/ADV-08_R.js
"use strict";
console.log("ADV-START");
const lines = require("fs").readFileSync(0, "utf8").split("\n");
const a = BigInt(lines[0]);
const b = BigInt(lines[1]);
const q = a / b;
console.log("OBS=QUOT:" + q);
console.log("ADV-END");
